features/fixtures/macos-stress-test/Makefile

```
@@ -19,3 +19,4 @@ run:
 	rm -rf $(HOME)/Library/Application\ Support/com.bugsnag.Bugsnag
 	QUIET=true /usr/bin/time -l ./build/usr/local/bin/BugsnagStressTest
 	rm -rf $(HOME)/Library/Application\ Support/com.bugsnag.Bugsnag
+	echo "MacOS stress-test complete"
```

nit: it's "macOS", not "MacOS" 😅 
```suggestion
	echo "macOS stress-test complete"
```